toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -334,3 +369,14 @@ func unbuiltPackagesCSVRows(pkgGraph *pkggraph.PkgGraph, unbuiltPackages, failed
 
 	return
 }
+
+// Helper function to return a sorted slice of a map's keys
+// Supports *pkggraph.PkgNode, bool and *BuildResult as map values
+func getSortedKeys[V *pkggraph.PkgNode | bool | *BuildResult](inputMap map[string]V) (keys []string) {
```

Comment:
We have some functions in `sliceutils`, we already have a function to convert a map to a slice at least. If we want a sort, maybe add it there?
Or, might be enough to just call `sort.Strings(sliceutils.SetToSlice(<set>))`

Reply:
The current function in sliceutils was returning a slice of keys, where the value of the key was True (bool). Added another generic function in sliceutils, which returns a slice of all keys.